tests/components/sensibo/conftest.py

```diff
@@ -62,7 +62,7 @@ async def get_data_from_library(
     return output
 
 
-@pytest.fixture(name="load_json", scope="session")
+@pytest.fixture(name="load_json")
```

Comment:
Separate this fixture into two fixtures. One that loads the data from disk and returns a string. This can be scoped as session. Then another that loads the string data into a json dict. That should be scoped per test to guard for mutations of the dict.